dashboard/test/ui/features/step_definitions/eyes_steps.rb

```diff
@@ -61,3 +69,12 @@ def ensure_eyes_available
   # BrowserStack was reporting Windows 6.0 and 6.1, causing different baselines
   @eyes.host_os = ENV['APPLITOOLS_HOST_OS']
 end
+
+# Wraps any http or https url text in an <a> tag pointing to the same url.
+# Only needs to work on urls like:
+# https://eyes.applitools.com/app/batches/00112233/44556677?accountId=abcdefg.
+# Note that applitools strips the trailing dot on this url for us, so we
+# don't need to strip it here.
+def linkify(str)
+  str.to_s.gsub(/(https?:\/\/[^ ]+)/, '<a href="\1">\1</a>')
```

Comment:
I'd replace `[^ ]` with `\S` so any whitespace ends the link.  Alternatively, there are a number of gems for this sort of thing.